src/features/goblins/storageHouse/components/StorageItems.tsx

```diff
@@ -41,6 +59,44 @@ export const StorageItems: React.FC = () => {
             );
           })}
         </div>
+
+        <p className="mb-2 underline">Goblin Community Treasury</p>
+        {loading ? (
+          <div className="my-2">Loading...</div>
+        ) : (
+          <div
+            className="flex flex-wrap h-fit -ml-1.5 mb-2"
+            style={{ minHeight: "150px" }}
```

Comment:
What is the purpose of the min height here? It seems to be causing some extra padding between the 2 rows of items.

<img width="367" alt="Screen Shot 2022-07-20 at 3 46 20 pm" src="https://user-images.githubusercontent.com/11745561/179906000-a5c95c3e-75da-49b2-826a-1b1bab34a596.png">

Reply:
Copied from Farm Storage div. Now removed.